packages/react-devtools-core/src/backend.js

```diff
@@ -60,6 +63,7 @@ export function initialize(
   maybeSettingsOrSettingsPromise?:
     | DevToolsHookSettings
     | Promise<DevToolsHookSettings>,
+  reloadAndProfileConfig: ReloadAndProfileConfig,
```

Comment:
looks like this is unused

Reply:
This is passed in from setUpReactDevTools.js https://www.internalfb.com/diff/D63233256?permalink=3800761013473689

Reply:
Yeah, but this is unused in this function.

You are passing this in `connectToDevTools` and in `connectWithCustomMessagingProtocol`, and you are using it there

Reply:
We're using it in all three places. `initialize` needs the initial (heh) boolean values to pass onto `renderer`, and the `connect*` functions need the full `*ConfigPersistence` to read & write the flags